<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>802</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" rowspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>试剂规格</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox1">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label1">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>条码类别：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit6">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label6">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>销售区域：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label0">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>仪器型号：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>规格：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cBoxCategory1">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>-请选择条码类别-</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>试剂A液</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>试剂B液</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>试剂C液</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>试剂H液</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>层析柱</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QTextEdit" name="textEditEncry">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label7">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>条码数目：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cBoxCategory0">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>-----请选择-----</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LD560</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LD600</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>试剂批号：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit4">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="btnGenerateCode">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>生成条码</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnClearCode">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>清空条码</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>层析柱：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label4">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>初始条码：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>内置密钥</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>自定义密钥</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEditKey">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QGroupBox" name="groupBox3">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>加密条码</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QPushButton" name="btnEncryption">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>加密</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnSave">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnSaveAs">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>另存为</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="btnClearEncry">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btnPrint">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>打印</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QPlainTextEdit" name="textEncryption">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>条码</string>
    </property>
    <addaction name="actionCheck"/>
    <addaction name="actioncode"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInfo">
   <property name="text">
    <string>信息</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>查看</string>
   </property>
  </action>
  <action name="actioncode">
   <property name="text">
    <string>校准条码</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
